src/js/components/Tabs/Tabs.js

```diff
@@ -39,6 +57,74 @@ const Tabs = forwardRef(
     delete rest.onActive;
     /* eslint-enable no-param-reassign */
 
+    const tabRefs = React.Children.map(children, () => React.createRef());
+
+    // check if tab is in view
+    function isVisible(element) {
+      const tabRect = element?.getBoundingClientRect();
+      const headerRect = targetRef.current?.getBoundingClientRect();
+      return (
+        tabRect.left >= headerRect.left - 1 &&
```

Comment:
Why do we need the -1 and +1?

Reply:
I found that I needed to add the -1 and +1 so that there was a little leniency when calculating if an element was in view. Without the -1 and +1 a tab could be fully in view but `isVisible` would return false.

Reply:
Added a comment in the code about this

Reply:
Not sure if this would achieve the same thing but what if instead of `>=`, we just do `>`. So it'd be:

`tabRect.left > headerRect.left`?

Or does that cause the same issue.

Reply:
unfortunately `tabRect.left > headerRect.left` does cause the same issue